packages/wallet-lib/src/types/WalletStore/methods/exportState.js

```diff
@@ -10,6 +11,7 @@ function exportState(chainHeight) {
   }
 
   return {
+    walletId,
```

Comment:
I think we don't need it there. Wallet id is already known from the root object. And at importState as well